packages/compass-components/src/components/leafygreen.tsx

```diff
@@ -49,25 +49,49 @@ import {
 import { Tabs as LeafyGreenTabs, Tab } from '@leafygreen-ui/tabs';
 import { default as LeafyGreenTextArea } from '@leafygreen-ui/text-area';
 import { default as LeafyGreenTextInput } from '@leafygreen-ui/text-input';
-import { default as Toast } from '@leafygreen-ui/toast';
+import {
+  default as Toast,
+  Variant as ToastVariant,
+} from '@leafygreen-ui/toast';
 import { default as LeafyGreenToggle } from '@leafygreen-ui/toggle';
-import { default as LeafyGreenTooltip } from '@leafygreen-ui/tooltip';
 import {
-  H1,
-  H2,
-  H3,
-  Subtitle,
-  Body,
-  InlineCode,
-  InlineKeyCode,
-  Disclaimer,
-  Overline,
+  H1 as LeafyGreenH1,
```

Comment:
Remove Tooltip re-wiring to avoid confusion with internal imports (now is done in `tooltip.ts` directly) and wire the theme to typography.
This is important now: tooltips in the rebranded LG have a dark background with a light mode, so typography elements must have an inverted theme to be visible.